soc/arm/nordic_nrf/common/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library_sources_ifdef(CONFIG_SOC_FAMILY_NRF soc_nrf_common.S)
+zephyr_include_directories(.)
```

Comment:
Might be a bit cleaner to move `../soc_nrf_common.h` to this directory, and `include <soc_nrf_common.h>` from each of the `../nrf*/soc.h` files, if you're adding a new common include directory here. Just an idea; feel free to ignore.

Reply:
done in a new commit